changelog/dmd.ftime-trace.dd

```diff
@@ -11,6 +11,6 @@ This will output `app.o.time-trace`.
 
 A different output file can be selected with `-ftime-trace-file=trace.json`.
 
-The output is in Google Chrome's profiler format, which can be converted to readable text with the included `timetrace2txt` tool, or be viewed in an interactive viewer like [ui.perfetto.dev](https://ui.perfetto.dev).
```

Comment:
I think the timetrace2txt tool is a necessity for practical use. Traces get very large quite quickly, and then the browser-based visual tooling becomes pretty slow (last time I looked). `tracy` https://github.com/wolfpld/tracy does have good performance on large traces though.

Reply:
Yes, it probably needs to be added to https://github.com/dlang/tools and https://github.com/dlang/installer. My plan is to get this PR approved first and then follow up by adding timetrace2txt, since this PR is already pretty big.